pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,59 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, buildPythonApplication
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, libao, libpulseaudio, alsaLib
+, withFestival ? false # TODO: , festival-freebsoft-utils
+, withEspeak-ng ? true, espeak-ng, sonic, pcaudiolib
 , withPico ? true, svox
+, withIvona ? false, # TODO: , libdumbtts
 }:
 
-stdenv.mkDerivation rec {
+with lib;
+buildPythonApplication rec {
   name = "speech-dispatcher-${version}";
-  version = "0.8.5";
+  version = "0.8.8";
+  format = "other";
 
   src = fetchurl {
     url = "http://www.freebsoft.org/pub/projects/speechd/${name}.tar.gz";
-    sha256 = "18jlxnhlahyi6njc6l6576hfvmzivjjgfjyd2n7vvrvx9inphjrb";
+    sha256 = "1wvck00w9ixildaq6hlhnf6wa576y02ac96lp6932h3k1n08jaiw";
   };
 
-  buildInputs = [ intltool libtool glib dotconf libsndfile libao python3Packages.python ]
-             ++ lib.optional withEspeak espeak
-             ++ lib.optional withPico svox;
-  nativeBuildInputs = [ pkgconfig python3Packages.wrapPython ];
-
-  hardeningDisable = [ "format" ];
-
-  pythonPath = with python3Packages; [ pyxdg ];
+  nativeBuildInputs = [ pkgconfig autoreconfHook intltool itstool texinfo ];
+
+  propagatedBuildInputs = [ pyxdg ];
+
+  buildInputs = [ libtool glib dotconf libsndfile libao libpulseaudio alsaLib ]
```

Comment:
It would be better to disable the other back-ends by default. `libao` already takes care of pulseaudio support and who uses alsa nowadays.

Reply:
Done